CRM/Civixero/Invoice.php

```diff
@@ -389,6 +380,18 @@ protected function getContributionsRequiringPushUpdate($params, $limit) {
     }
 
     $records = civicrm_api3('AccountInvoice', 'get', $criteria);
+
+    $pushEnabledStatuses = Civi::settings()->get('xero_push_contribution_status');
```

Comment:
Actually - I was imagining combining the statues with this line

https://github.com/jitendrapurohit/nz.co.fuzion.civixero/blob/2f719b97a1cb14f5cbd36628c3032b25de69ac78/CRM/Civixero/Invoice.php#L371

We should probably have some defaults (Completed, Pending) & then not have the isNotCancelled thingee